sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/LimitPushdownSuite.scala

```diff
@@ -113,35 +113,34 @@ class LimitPushdownSuite extends PlanTest {
 
   test("full outer join where neither side is limited and both sides have same statistics") {
```

Comment:
Hi, @gatorsmile and @henryr .
This breaks `branch-2.2`.
```
[info] - full outer join where neither side is limited and both sides have same statistics *** FAILED *** (43 milliseconds)
[info] - full outer join where neither side is limited and left side has larger statistics *** FAILED *** (12 milliseconds)
[info] - full outer join where neither side is limited and right side has larger statistics *** FAILED *** (9 milliseconds)
```
cc @felixcheung for 2.2.1.

Reply:
Could you submit a fix to resolve the test failure?

Reply:
Sure, I'm taking a look now.

Reply:
Thank you, all!

Reply:
See https://github.com/apache/spark/pull/19701. I think something got dropped in the backport to branch-2.2, so thankfully it's an easy fix.